mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -2847,6 +2847,63 @@ func TestAccContainerCluster_withSandboxConfig(t *testing.T) {
 }
 {{- end }}
 
+func TestAccContainerCluster_withSandboxConfigType(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	networkName := acctest.BootstrapSharedTestNetwork(t, "gke-cluster")
+	subnetworkName := acctest.BootstrapSubnet(t, "gke-cluster", networkName)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withSandboxConfigType(clusterName, networkName, subnetworkName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_cluster.with_sandbox_config",
+						"node_config.0.sandbox_config.0.type", "GVISOR"),
+					resource.TestCheckResourceAttr("google_container_cluster.with_sandbox_config",
+						"node_pool.0.node_config.0.sandbox_config.0.type", "GVISOR"),
+				),
```

Comment:
FWIW, for some reason a lot of container tests have Checks, but they are basically duplicated by the ImportStateVerify step so usually aren't necessary. Not a blocker though.